src/Neo.CLI/CLI/MainService.Plugins.cs

```diff
@@ -85,26 +85,57 @@ private static async Task<Stream> DownloadPluginAsync(string pluginName, Version
 
             var asmName = Assembly.GetExecutingAssembly().GetName();
             httpClient.DefaultRequestHeaders.UserAgent.Add(new(asmName.Name!, asmName.Version!.ToString(3)));
+
             var url = customDownloadUrl == null ? Settings.Default.Plugins.DownloadUrl : new Uri(customDownloadUrl);
             var json = await httpClient.GetFromJsonAsync<JsonArray>(url) ?? throw new HttpRequestException($"Failed: {url}");
+            var pluginVersionString = $"v{pluginVersion.ToString(3)}";
+
             var jsonRelease = json.AsArray()
-                .SingleOrDefault(s =>
-                    s != null &&
-                    s["tag_name"]!.GetValue<string>() == $"v{pluginVersion.ToString(3)}" &&
-                    s["prerelease"]!.GetValue<bool>() == prerelease) ?? throw new Exception($"Could not find Release {pluginVersion}");
+                .FirstOrDefault(s =>
+                    s?["tag_name"]?.GetValue<string>() == pluginVersionString &&
+                    s["prerelease"]?.GetValue<bool>() == prerelease);
+
+            if (jsonRelease == null)
+            {
+                jsonRelease = json.AsArray()
+                    .Where(s => s?["prerelease"]?.GetValue<bool>() == prerelease)
+                    .Select(s =>
+                    {
+                        var tagName = s?["tag_name"]?.GetValue<string>();
+                        return Version.TryParse(tagName?[1..], out var version)
+                            ? new { JsonObject = s, Version = version }
+                            : null;
+                    })
+                    .OfType<dynamic>()
+                    .OrderByDescending(s => s.Version)
+                    .Select(s => s.JsonObject)
+                    .FirstOrDefault();
+
+                if (jsonRelease != null)
+                {
+                    var latestVersion = Version.Parse(jsonRelease["tag_name"]!.GetValue<string>()[1..]);
```

Comment:
Use `Version.TryParse` instead of `Version.Parse`?

Reply:
This comes from a webservice contract. So `Version.Parse` is good enough.